.github/workflows/static.yml

```diff
@@ -180,10 +180,10 @@ jobs:
           cd $GITHUB_WORKSPACE
           ./create-dev-env.sh
 
-      - name: "Migrate database to version 10"
+      - name: "Migrate database to version x"
         run: |
           cd $GITHUB_WORKSPACE/utilities/db-migration
-          go run main.go migrate_to 10
+          go run main.go migrate_to 11
```

Comment:
@jgwest this will make test to to fail as `error on inserting cluster user: ERROR #42703 column \"created_on\" of relation \"clusteruser\" does not exist` as created_on field for clusteruser is not present in version 10

Reply:
Got it, thanks @Rizwana777! Feel free to put it back to 11... I think I need to think more about how this migration PR check should work, but for this PR we can keep the migrate_to at 11in the mean time.